builder/builder-next/executor_unix.go

```diff
@@ -28,7 +28,7 @@ func newExecutor(root, cgroupParent string, net libnetwork.NetworkController, ro
 	}
 	return runcexecutor.New(runcexecutor.Opt{
 		Root:                filepath.Join(root, "executor"),
-		CommandCandidates:   []string{"runc"},
+		CommandCandidates:   []string{"runu", "runc"},
```

Comment:
can we use something variable here (e.g., `daemon.DefaultShimBinary`  as in cmd/dockerd/daemon_unix.go) ?

Reply:
This works: 	CommandCandidates:   []string{daemon.DefaultShimBinary},

Should I push this one in or needs more change?

Reply:
yes please.